pandas-stubs/core/indexes/multi.pyi

```diff
@@ -68,6 +69,8 @@ class MultiIndex(Index):
     def __contains__(self, key) -> bool: ...
     @property
     def dtype(self) -> np.dtype: ...
+    @property
+    def dtypes(self) -> pd.Series[Any]: ...
```

Comment:
When we can't infer the type of the `Series`, we just return `pd.Series`.  So drop the `Any` part.